lib/src/collections/observable_list.dart

```diff
@@ -25,6 +25,11 @@ import 'package:observable/src/differs.dart';
 ///
 /// *See [ListDiffer] to manually diff two lists instead*
 abstract class ObservableList<E> implements List<E>, Observable {
+  /// An empty observable list that never has changes.
+  static const ObservableList EMPTY = const _ObservableUnmodifiableList(
+    const [],
```

Comment:
Doesn't fit on one line unfortunately - this at least looks like the other files for now.